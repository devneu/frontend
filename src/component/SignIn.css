.SignInWrapper {
  width: 100%;
  height: 100vh;
  display: flex;
  justify-content: center;
  align-items: center;
}

.SignInBlockFlex {
  width: 60%;
  height: 500px;
  border-radius: 5px;
  display: flex;
  justify-content: center;
  align-items: center;
}

.ImgBlock {
  height: 100%;
  width: 50%;
  background-color: darkmagenta;
  border-radius: 20px 0px 0px 20px;
  display: flex;
  justify-content: center;
  align-items: center;
}
.ImgBlock img {
  width: 100%;
  height: 346.275px;
}

.FormBlock {
  height: 100%;
  text-align: center;
  width: 50%;
  background-color: lightseagreen;
  border-radius: 0px 20px 20px 0px;
  display: flex;
  justify-content: center;
  align-items: center;
}
.FormBlock form {
  display: flex;
  flex-direction: column;
  align-items: center;
}
.FormBlock form input, .FormBlock form button {
  margin: 5px 0px;
}
.FormBlock form hr {
  width: 100%;
}

.errorMassage {
  color: darkred;
  text-align: left;
  padding: 2px 2px;
  font-size: 13px;
  width: 50%;
  background-color: darkorange;
  border-radius: 5px;
}

/*# sourceMappingURL=SignIn.css.map */
