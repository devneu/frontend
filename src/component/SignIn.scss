%flex-template {
  display: flex;
  justify-content: center;
  align-items: center;
}

.sign-in-wrapper {
  width: 100%;
  height: 100vh;
  @extend %flex-template;

  .sign-in-block-flex {
    width: 60%;
    height: 500px;
    border-radius: 5px;
    @extend %flex-template;


    .img-block {
      height: 100%;
      width: 50%;
      background-color: darkmagenta;
      border-radius: 20px 0px 0px 20px;
      @extend %flex-template;

      img {
        width: 100%;
        height: 346.275px;
      }
    }


    .form-block {
      height: 100%;
      text-align: center;
      width: 50%;
      background-color: lightseagreen;
      border-radius: 0px 20px 20px 0px;
      @extend %flex-template;

      form {
        display: flex;
        flex-direction: column;
        align-items: center;

        input, button {
          margin: 5px 0px;
        }

        hr {
          width: 100%;
        }
      }
    }
  }
}

.error-massage {
  color: darkred;
  text-align: left;
  padding: 2px 2px;
  font-size: 13px;
  width: 50%;
  background-color: darkorange;
  border-radius: 5px;
}