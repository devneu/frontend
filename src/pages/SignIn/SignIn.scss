$border-size: 5px 0px 0px 5px;
$width: 100%;

.sign-in {
  padding: 20px 0px;
  background-color: #0050b3;

  .img-block {
    border-radius: $border-size;


    img {
      width: $width;
      height: 100%;
      object-fit: cover;
      object-position: center;
      border-radius: $border-size;
    }
  }

  .form-block {
    background-color: white;
    border-radius: 0px 5px 5px 0px;
    display: flex;
    justify-content: center;
    align-items: center;

    form {

      .title-form {
        text-align: center;
        margin-top: 20px;
      }

      input, button {
        margin: 5px 0px;
      }

      .line {
        height: 0.1px;
        background-color: lightgray;
        border: none;
      }

      .email, .password {
        border-radius: 20px;
      }

      .error-message {
        color: white;
        padding: 2px 2px;
        margin: 5px;
        font-size: 13px;
        width: $width;
        border-radius: 5px;
        background-color: #f5222d;
        text-align: center;
      }


      .ant-btn-dangerous {
        background-color: #f5222d;
        color: white;

        &:hover {
          background-color: #cf1322;
        }
      }

      .facebook-btn {
        background-color: #0050b3;
        border: none;
        height: 32px;
        padding: 4px 16px;
        border-radius: 32px;
        width: $width;
        color: white;
        transition: all .5s ease;
        font-size: 13px;

        &:hover {
          background-color: #003a8c;
          cursor: pointer;
        }

        i {
          margin-right: 5px;
        }
      }

      .footer {
        text-align: center;
        margin-bottom: 20px;
      }
    }
  }
}












