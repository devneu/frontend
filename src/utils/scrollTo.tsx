import { scroller } from 'react-scroll';

function scrollTo(className: string): void {
  scroller.scrollTo(className, {
    duration: 300,
    delay: 0,
    offset: -100,
    smooth: 'easeInOutQuart',
  });
}

export default scrollTo;
