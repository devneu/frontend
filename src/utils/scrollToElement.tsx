import { scroller } from "react-scroll";

function scrollToElement(className: string): void {
   scroller.scrollTo(className, {
      duration: 300,
      delay: 0,
      offset: -100,
      smooth: "easeInOutQuart",
   })
}

export default scrollToElement;