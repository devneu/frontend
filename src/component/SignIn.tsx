import React from "react";
import "./SignIn.scss";
import logo from "../logo.svg";
import {useForm, SubmitHandler} from "react-hook-form";
import GoogleLogin from 'react-google-login';
import FacebookLogin from 'react-facebook-login';

type Inputs = {
    email: string,
    password: string,
    checkBoxflag: boolean
};


const SignIn = () => {
    const avaibleSymb = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;

    const {
        register,
        handleSubmit,
        formState: {errors}
    } = useForm();

    const onSubmit: SubmitHandler<Inputs> = data => console.log("login button basic: ", data);

    const responseGoogle = (response: any) => {
        console.log("google profileInfo: ", response.profileObj);
    }
    const responseFacebook = (response: any) => {
        console.log("facebook profileInfo: ", response);
    }


    return (
        <div className="sign-in-wrapper">
            <div className="sign-in-block-flex">
                <div className="img-block">
                    <img src={logo}/>
                </div>
                <div className="form-block">
                    <form onSubmit={handleSubmit(onSubmit)}>
                        <h1>Hello !</h1>

                        <input type="text" placeholder="Enter your mail..." {...register("email", {required: true, maxLength: 50, pattern: avaibleSymb})}/>
                        {errors.email && <span className="error-massage"><i>This input is required , maximum length is 50 symbols and has such template as: expample.exam@gmail.com</i></span>}

                        <input type="password" placeholder="Password" {...register("password", {required: true})}/>
                        {errors.password && <span className="error-massage"><i>This input is required</i></span>}

                        <label><input type="checkbox" {...register("checkBoxflag")}/>Remember me</label>
                        <input type="submit" value="login"/>

                        <hr/>

                        <GoogleLogin
                            clientId="459389646319-e4g64jlp1jfpi66h3kgv1uuek71la5b0.apps.googleusercontent.com"
                            buttonText="Login"
                            onSuccess={responseGoogle}
                            onFailure={responseGoogle}
                            cookiePolicy={'single_host_origin'}
                        />

                        <FacebookLogin
                            appId="226836885848996"
                            autoLoad={false}
                            fields="name,email,picture"
                            callback={responseFacebook}
                            cssClass="my-facebook-button-class"
                            icon="fa-facebook"
                        />

                        <hr/>

                        <a href="#">forgot password</a>
                        <a href="#">Create account</a>
                    </form>
                </div>
            </div>
        </div>
    );
}
export default SignIn;