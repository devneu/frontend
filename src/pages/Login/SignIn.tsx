import React from "react";
import "./SignIn.scss";
import {Form, Input, Button, Checkbox, Row, Col, Typography} from 'antd';
import {useForm, SubmitHandler, Controller} from "react-hook-form";
import GoogleLogin from 'react-google-login';
import FacebookLogin from 'react-facebook-login';
import {GoogleOutlined} from "@ant-design/icons";
const { Title, Link } = Typography;

type Inputs = {
    email: string,
    password: string,
    checkBoxflag: boolean
};


const SignIn = () => {
    const avaibleSymb = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;

    const {
        register,
        handleSubmit,
        formState: {errors},
        control
    } = useForm();

    const onSubmit = (data: any) => console.log("Login basic", data);

    function onChange(e: any) {
        console.log(`checked = ${e.target.checked}`);
    }

    const responseGoogle = (response: any) => {
        console.log("google profileInfo: ", response.profileObj);
    }
    const responseFacebook = (response: any) => {
        console.log("facebook profileInfo: ", response);
    }


    return (
        <div className="SignInWrapper">
            <Row justify="center">
                <Col className="ImgBlock" xs={0} sm={0} md={10} lg={10} xl={12}>
                    <img src="https://source.unsplash.com/K4mSJ7kc0As/600x800"/>
                </Col>
                <Col className="FormBlock" xs={20} sm={22} md={10} lg={10} xl={12}>
                    <Col span = {18}>
                        <Form onFinish={handleSubmit(onSubmit)}>

                            <Title className = "title-form" level={2}>Welcome Back!</Title>

                            <Form.Item name="email">
                                <Input className="email" placeholder="Email" {...register("email", {required: true, maxLength: 50, pattern: avaibleSymb})}></Input>
                                {errors.email && <div className="errorMassage"><span>This input is required , maximum length is 50 symbols and has such template as: expample.exam@gmail.com</span></div>}
                            </Form.Item>

                            <Form.Item name="password">
                                <Input.Password className="password" placeholder="Password" {...register("password", {required: true})}></Input.Password>
                                {errors.password && <div className="errorMassage"><span>This input is required</span></div>}
                            </Form.Item>

                            <Controller
                                control={control}
                                name="remeberMe"
                                defaultValue={false}
                                render={({
                                             field: { onChange, onBlur, value, name, ref },
                                             fieldState: { invalid, isTouched, isDirty, error },
                                             formState,
                                         }) => (
                                    <Checkbox
                                        onChange={onChange}
                                        checked={value}
                                    >Remember Me</Checkbox>
                                )}
                            />



                            <Form.Item>
                                <Button block size = "large" type="primary" htmlType="submit" shape="round">login</Button>
                            </Form.Item>

                            <hr className = "line"/>

                            <Form.Item>
                                <GoogleLogin
                                    clientId="459389646319-e4g64jlp1jfpi66h3kgv1uuek71la5b0.apps.googleusercontent.com"
                                    buttonText="Login"
                                    onSuccess={responseGoogle}
                                    onFailure={responseGoogle}
                                    cookiePolicy={'single_host_origin'}
                                    render={renderProps => (
                                        <Button onClick={renderProps.onClick} block danger size = "large" htmlType="submit"
                                                shape="round"><GoogleOutlined/>Login with Google</Button>
                                    )}
                                />
                            </Form.Item>

                            <FacebookLogin
                                appId="226836885848996"
                                autoLoad={false}
                                fields="name,email,picture"
                                callback={responseFacebook}
                                cssClass="my-facebook-button-class"
                                icon="fa-facebook"
                            />



                            <hr className = "line"/>

                            <div className="footer-link">
                                <br/>
                                <Link href="/new-pass" target="_blank">
                                    Forgot password ?
                                </Link>
                                <br/>
                                <Link href="/create-log" target="_blank">
                                    Create an Account!
                                </Link>
                            </div>

                        </Form>
                    </Col>
                </Col>
            </Row>
        </div>
    );
}
export default SignIn;

/*
<Form onFinish={handleSubmit(onSubmit)}>

    <Form.Item name="email">
        <Input className="email" placeholder="Email" {...register("email", {required: true, maxLength: 50, pattern: avaibleSymb})}></Input>
        {errors.email &&
        <span className="errorMassage"><i>This input is required , maximum length is 50 symbols and has such template as: expample.exam@gmail.com</i></span>}
    </Form.Item>

    <Form.Item name="password">
        <Input.Password className="password" placeholder="Password" {...register("password", {required: true})}></Input.Password>
        {errors.password && <span className="errorMassage"><i>This input is required</i></span>}
    </Form.Item>

    <Form.Item valuePropName="checked">
        <Checkbox {...register("remember-me")} >Remember Me</Checkbox>
    </Form.Item>

    <Form.Item>
        <Button block type="primary" htmlType="submit" shape="round">login</Button>
    </Form.Item>

    <hr/>

    <Form.Item>
        <GoogleLogin
            clientId="459389646319-e4g64jlp1jfpi66h3kgv1uuek71la5b0.apps.googleusercontent.com"
            buttonText="Login"
            onSuccess={responseGoogle}
            onFailure={responseGoogle}
            cookiePolicy={'single_host_origin'}
            render={renderProps => (
                <Button onClick={renderProps.onClick} block danger htmlType="submit"
                        shape="round"><GoogleOutlined/>Login with Google</Button>
            )}
        />
    </Form.Item>

    <FacebookLogin
        appId="226836885848996"
        autoLoad={false}
        fields="name,email,picture"
        callback={responseFacebook}
        cssClass="my-facebook-button-class"
        icon="fa-facebook"
    />
    <hr/>

    <div className="footer-link">
        <br/>
        <a href="#">Forgot Password?</a>
        <br/>
        <a href="#">Create an Account!</a>
    </div>

</Form>*/
