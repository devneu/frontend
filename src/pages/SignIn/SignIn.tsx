import React from "react";
import "./SignIn.scss";
import {EMAIL_PATTERN} from "../../utils/formValidations";
import FormErrorMessage from "../../components/FormErrorMessage/FormErrorMassage";
import {Form, Input, Button, Checkbox, Row, Col, Typography} from 'antd';
import {useForm, SubmitHandler, Controller} from "react-hook-form";
import GoogleLogin from 'react-google-login';
import FacebookLogin from 'react-facebook-login';
import {GoogleOutlined} from "@ant-design/icons";
const { Title, Link } = Typography;


const SignIn = () => {

    const {
        register,
        handleSubmit,
        formState: {errors},
        control
    } = useForm();

    const onSubmit = (data: any) => console.log("SignIn basic", data);

    const handleGoogleResponse  = (response: any) => {
        console.log("google profileInfo: ", response.profileObj);
    }
    const handleFacebookResponse  = (response: any) => {
        console.log("facebook profileInfo: ", response);
    }


    return (
        <div className="sign-in">
            <Row justify="center">


                <Col className="img-block" xs={0} sm={0} md={10} lg={10} xl={9} xxl={8}>
                    <img src="https://source.unsplash.com/K4mSJ7kc0As/600x800"/>
                </Col>



                <Col className="form-block" xs={20} sm={22} md={10} lg={10} xl={9} xxl={8}>
                    <Col span={18}>


                        <Form onFinish={handleSubmit(onSubmit)}>


                            <Title className="title-form" level={2}>Welcome Back!</Title>


                            <Form.Item name="email">
                                <Input className="email" placeholder="Email" {...register("email", {required: "This input is required , maximum length is 50 symbols and has such template as: expample.exam@gmail.com", maxLength: 50, pattern: EMAIL_PATTERN})}></Input>
                                <FormErrorMessage message={errors.email?.message}/>
                            </Form.Item>

                            <Form.Item name="password">
                                <Input.Password className="password" placeholder="Password" {...register("password", {required: "This input is required"})}></Input.Password>
                                <FormErrorMessage message={errors.password?.message}/>
                            </Form.Item>

                            <Controller
                                control={control}
                                name="remeberMe"
                                defaultValue={false}
                                render={({
                                             field: { onChange, value },
                                         }) => (
                                    <Checkbox
                                        onChange={onChange}
                                        checked={value}
                                    >Remember Me</Checkbox>
                                )}
                            />

                            <Form.Item>
                                <Button block type="primary" htmlType="submit" shape="round">Login</Button>
                            </Form.Item>

                            <hr className="line"/>

                            <Form.Item>
                                <GoogleLogin
                                    clientId="459389646319-e4g64jlp1jfpi66h3kgv1uuek71la5b0.apps.googleusercontent.com"
                                    buttonText="Login"
                                    onSuccess={handleGoogleResponse }
                                    onFailure={handleGoogleResponse }
                                    cookiePolicy={'single_host_origin'}
                                    render={renderProps => (
                                        <Button onClick={renderProps.onClick} block danger htmlType="submit"
                                                shape="round"><GoogleOutlined/>Login with Google</Button>
                                    )}
                                />
                            </Form.Item>

                            <FacebookLogin
                                appId="226836885848996"
                                autoLoad={false}
                                fields="name,email,picture"
                                callback={handleFacebookResponse}
                                cssClass="my-facebook-button-class"
                                icon="fa-facebook"
                            />

                            <hr className="line"/>


                            <div className="footer">
                                <br/>
                                <Link href="/forgot-password">
                                    Forgot password ?
                                </Link>
                                <br/>
                                <Link href="/sign-out">
                                    Create an Account!
                                </Link>
                            </div>

                        </Form>


                    </Col>
                </Col>

            </Row>
        </div>
    );
}
export default SignIn;

