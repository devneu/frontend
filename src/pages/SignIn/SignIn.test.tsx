import {matchMediaCaller} from "../../utils/setupjest";
import renderer from 'react-test-renderer';
import SignIn from './SignIn';

matchMediaCaller();

jest.mock('react-facebook-login', () => () => 'MockFacebookLogin');

describe('SignIn', () => {
    it('Should successfully render', () => {
        const wrapper = renderer
            .create(
                <SignIn />
            )
            .toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});
