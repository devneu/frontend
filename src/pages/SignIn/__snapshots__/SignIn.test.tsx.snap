// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn Should successfully render 1`] = `
<div
  className="sign-in"
>
  <div
    className="ant-row ant-row-center"
    style={Object {}}
  >
    <div
      className="ant-col img-block ant-col-xs-0 ant-col-sm-0 ant-col-md-10 ant-col-lg-10 ant-col-xl-9 ant-col-xxl-8"
      style={Object {}}
    >
      <img
        src="https://source.unsplash.com/K4mSJ7kc0As/600x800"
      />
    </div>
    <div
      className="ant-col form-block ant-col-xs-20 ant-col-sm-22 ant-col-md-10 ant-col-lg-10 ant-col-xl-9 ant-col-xxl-8"
      style={Object {}}
    >
      <div
        className="ant-col ant-col-18"
        style={Object {}}
      >
        <form
          className="ant-form ant-form-horizontal"
          onReset={[Function]}
          onSubmit={[Function]}
        >
          <h2
            className="ant-typography title-form"
            style={
              Object {
                "WebkitLineClamp": undefined,
              }
            }
          >
            Welcome Back!
          </h2>
          <div
            className="ant-row ant-form-item"
            style={Object {}}
          >
            <div
              className="ant-col ant-form-item-control"
              style={Object {}}
            >
              <div
                className="ant-form-item-control-input"
              >
                <div
                  className="ant-form-item-control-input-content"
                >
                  <input
                    className="ant-input email"
                    name="email"
                    onBlur={[Function]}
                    onChange={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    placeholder="Email"
                    type="text"
                    value=""
                  />
                  
                </div>
              </div>
            </div>
          </div>
          <div
            className="ant-row ant-form-item"
            style={Object {}}
          >
            <div
              className="ant-col ant-form-item-control"
              style={Object {}}
            >
              <div
                className="ant-form-item-control-input"
              >
                <div
                  className="ant-form-item-control-input-content"
                >
                  <span
                    className="ant-input-affix-wrapper ant-input-password password"
                    onMouseUp={[Function]}
                  >
                    <input
                      action="click"
                      className="ant-input"
                      name="password"
                      onBlur={[Function]}
                      onChange={[Function]}
                      onFocus={[Function]}
                      onKeyDown={[Function]}
                      placeholder="Password"
                      style={null}
                      type="password"
                      value=""
                    />
                    <span
                      className="ant-input-suffix"
                    >
                      <span
                        aria-label="eye-invisible"
                        className="anticon anticon-eye-invisible ant-input-password-icon"
                        onClick={[Function]}
                        onMouseDown={[Function]}
                        onMouseUp={[Function]}
                        role="img"
                        tabIndex={-1}
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="eye-invisible"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M942.2 486.2Q889.47 375.11 816.7 305l-50.88 50.88C807.31 395.53 843.45 447.4 874.7 512 791.5 684.2 673.4 766 512 766q-72.67 0-133.87-22.38L323 798.75Q408 838 512 838q288.3 0 430.2-300.3a60.29 60.29 0 000-51.5zm-63.57-320.64L836 122.88a8 8 0 00-11.32 0L715.31 232.2Q624.86 186 512 186q-288.3 0-430.2 300.3a60.3 60.3 0 000 51.5q56.69 119.4 136.5 191.41L112.48 835a8 8 0 000 11.31L155.17 889a8 8 0 0011.31 0l712.15-712.12a8 8 0 000-11.32zM149.3 512C232.6 339.8 350.7 258 512 258c54.54 0 104.13 9.36 149.12 28.39l-70.3 70.3a176 176 0 00-238.13 238.13l-83.42 83.42C223.1 637.49 183.3 582.28 149.3 512zm246.7 0a112.11 112.11 0 01146.2-106.69L401.31 546.2A112 112 0 01396 512z"
                          />
                          <path
                            d="M508 624c-3.46 0-6.87-.16-10.25-.47l-52.82 52.82a176.09 176.09 0 00227.42-227.42l-52.82 52.82c.31 3.38.47 6.79.47 10.25a111.94 111.94 0 01-112 112z"
                          />
                        </svg>
                      </span>
                    </span>
                  </span>
                  
                </div>
              </div>
            </div>
          </div>
          <label
            className="ant-checkbox-wrapper"
          >
            <span
              className="ant-checkbox"
              style={Object {}}
            >
              <input
                checked={false}
                className="ant-checkbox-input"
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyPress={[Function]}
                onKeyUp={[Function]}
                type="checkbox"
              />
              <span
                className="ant-checkbox-inner"
              />
            </span>
            <span>
              Remember Me
            </span>
          </label>
          <div
            className="ant-row ant-form-item"
            style={Object {}}
          >
            <div
              className="ant-col ant-form-item-control"
              style={Object {}}
            >
              <div
                className="ant-form-item-control-input"
              >
                <div
                  className="ant-form-item-control-input-content"
                >
                  <button
                    className="ant-btn ant-btn-primary ant-btn-round ant-btn-block"
                    onClick={[Function]}
                    type="submit"
                  >
                    <span>
                      Login
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
          <hr
            className="line"
          />
          <div
            className="ant-row ant-form-item"
            style={Object {}}
          >
            <div
              className="ant-col ant-form-item-control"
              style={Object {}}
            >
              <div
                className="ant-form-item-control-input"
              >
                <div
                  className="ant-form-item-control-input-content"
                >
                  <button
                    className="ant-btn ant-btn-round ant-btn-block ant-btn-dangerous"
                    onClick={[Function]}
                    type="submit"
                  >
                    <span
                      aria-label="google"
                      className="anticon anticon-google"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="google"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="64 64 896 896"
                        width="1em"
                      >
                        <path
                          d="M881 442.4H519.7v148.5h206.4c-8.9 48-35.9 88.6-76.6 115.8-34.4 23-78.3 36.6-129.9 36.6-99.9 0-184.4-67.5-214.6-158.2-7.6-23-12-47.6-12-72.9s4.4-49.9 12-72.9c30.3-90.6 114.8-158.1 214.7-158.1 56.3 0 106.8 19.4 146.6 57.4l110-110.1c-66.5-62-153.2-100-256.6-100-149.9 0-279.6 86-342.7 211.4-26 51.8-40.8 110.4-40.8 172.4S151 632.8 177 684.6C240.1 810 369.8 896 519.7 896c103.6 0 190.4-34.4 253.8-93 72.5-66.8 114.4-165.2 114.4-282.1 0-27.2-2.4-53.3-6.9-78.5z"
                        />
                      </svg>
                    </span>
                    <span>
                      Login with Google
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
          MockFacebookLogin
          <hr
            className="line"
          />
          <div
            className="footer"
          >
            <br />
            <a
              className="ant-typography"
              href="/forgot-password"
              style={
                Object {
                  "WebkitLineClamp": undefined,
                }
              }
            >
              Forgot password ?
            </a>
            <br />
            <a
              className="ant-typography"
              href="/sign-out"
              style={
                Object {
                  "WebkitLineClamp": undefined,
                }
              }
            >
              Create an Account!
            </a>
          </div>
        </form>
      </div>
    </div>
  </div>
</div>
`;
